axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -630,6 +630,9 @@
       "horizonRpc": "https://horizon-testnet.stellar.org",
       "networkType": "testnet",
       "chainType": "stellar",
+      "decimals": 7,
+      "finality": "1",
```

Comment:
@blockchainguyy Can you share your research on the finality mechanism? Since it's a critical piece, we should validate

Reply:
@blockchainguyy Can you confirm whether "transactions are finalized in about 5 secs" means that as soon as they're included in a ledger, it's final, or is there some delay after a ledger is created (queryable in the RPC)?